lisa/util/shell.py

```diff
@@ -47,8 +47,9 @@ def wait_tcp_port_ready(
                     if times % 10 == 0 and log:
                         log.debug(
                             f"cannot connect to {address}:{port}, "
-                            f"error code: {result}, tried times: {times},"
-                            f" elapsed: {timout_timer}. retrying..."
+                            f"error : {os.strerror(result)}, current try: {times + 1},"
```

Comment:
Can it still print 10060 and 10061? They are important to see if it's none respond or denied by firewall.

Reply:
Yeah, unfortunately that not seems to be multi-platform and I declare timeout in finding something that would work for both. Revert that detail here. Thanks.